clients/src/main/java/org/apache/kafka/clients/admin/ListConsumerGroupOffsetsResult.java

```diff
@@ -34,7 +34,7 @@ public class ListConsumerGroupOffsetsResult {
 
     final KafkaFuture<Map<TopicPartition, OffsetAndMetadata>> future;
 
-    ListConsumerGroupOffsetsResult(KafkaFuture<Map<TopicPartition, OffsetAndMetadata>> future) {
+    public ListConsumerGroupOffsetsResult(KafkaFuture<Map<TopicPartition, OffsetAndMetadata>> future) {
```

Comment:
We need to push this public in order to be used in admin client callers.

Reply:
We usually avoid making those public. Given that it is only used in tests, I wonder if we could add a factory method in `AdminClientTestUtils` and keep it package private. What do you think?

Reply:
Good idea! Will do